html/elements/script.json

```diff
@@ -551,7 +551,7 @@
             "status": {
               "experimental": false,
               "standard_track": true,
-              "deprecated": true
+              "deprecated": false
```

Comment:
This is done in #7264. I think we should instead create `type.javascript` and mark it deprecated, because `type="module"` is a totally modern and valid thing.